.github/workflows/stalled.yml

```diff
@@ -0,0 +1,28 @@
+name: Label Stalled PRs
+on:
+  schedule:
+    - cron: '15 15 * * *' # Run every day at 15:15 UTC / 7:15 PST / 8:15 PDT
```

Comment:
Can you pick different time to avoid running the same process across repos at the same time? I believe those github actions are using shared resources.

Reply:
@Swiddis do you have other PR's open?

Reply:
It shouldn't make much of a dent compared to the storm of actions that run at midnight UTC (or the dozens of other CI actions running at any given time), if action runners aren't available at the scheduled time they'll just be delayed a couple minutes until a runner is free.